seatunnel-connectors-v2/connector-http/connector-http-base/src/main/java/org/apache/seatunnel/connectors/seatunnel/http/config/HttpParameter.java

```diff
@@ -43,9 +43,10 @@ public void buildWithConfig(Config pluginConfig) {
         this.setUrl(pluginConfig.getString(HttpConfig.URL.key()));
         // set method
         if (pluginConfig.hasPath(HttpConfig.METHOD.key())) {
-            this.setMethod(pluginConfig.getString(HttpConfig.METHOD.key()));
+            Method method = Method.valueOf(pluginConfig.getString(HttpConfig.METHOD.key()).toUpperCase());
+            this.setMethod(method);
         } else {
-            this.setMethod(HttpConfig.METHOD_DEFAULT_VALUE);
+            this.setMethod(Method.GET);
```

Comment:
`HttpConfig.METHOD.defaultValue()` is better.

Reply:
Thanks. I have fixed it.